extensions/sql-bindings/package.json

```diff
@@ -51,7 +51,7 @@
         },
         {
           "command": "sqlBindings.createAzureFunction",
-          "when": "view == objectExplorer && viewItem == Table && !azdataAvailable",
+          "when": "!azdataAvailable",
```

Comment:
This will show in the command palette as an option for the user to use only in vscode currently.